test/e2e/components/vl-cookie-consent-opt-in.js

```diff
@@ -0,0 +1,20 @@
+const { VlElement } = require('vl-ui-core');
+const { VlCheckbox } = require('vl-ui-checkbox');
+const { By } = require('selenium-webdriver');
+
+class VlCookieConsentOptIn extends VlElement {
+    async _getCheckbox() {
+        const checkboxElement = await this.shadowRoot.findElement(By.css('vl-checkbox'));
+        return new VlCheckbox(this.driver, checkboxElement);
+    }
+
+    async isOptedIn() {
+        return (await this._getCheckbox()).isChecked();
+    }
+
+    async optIn() {
```

Comment:
ik denk dat ge dit ofwel "toggle" moet noemen, ofwel dat ge 2 methodes optIn en optOut nodig hebt waarin ge eerst checkt wat de huidige value van de checkbox is ...
ik zou persoonlijk eerder het 2e kiezen.

Reply:
Akkoord. Opgesplitst in twee functies die telkens eerst de state van de checkbox controleren en naargelang de state al dan niet actie ondernemen.